Source/WarpX.H

```diff
@@ -1382,6 +1388,10 @@ private:
     int noy_fft = 16;
     int noz_fft = 16;
 
+    int nox_loc_fft = 8;
+    int noy_loc_fft = 8;
+    int noz_loc_fft = 8;
+
```

Comment:
As discussed offline, let's have default values such as the default asymmetrical behavior actually matches the symmetrical PSATD.

Reply:
Agreed and fixed.